kolibri/plugins/setup_wizard/assets/src/views/SelectFacilityForm.vue

```diff
@@ -99,6 +117,10 @@
         return this.fetchNetworkLocationFacilities(deviceId)
           .then(data => {
             this.facilities = [...data.facilities];
+            if (this.facilities.length === 0) {
+              this.errorMessage = SelectFacilityStrings.$tr('noFacilitiesText');
+              return;
+            }
```

Comment:
With the devices filtered in the modal, this should now be dead code (unreachable), right?

Reply:
_Should_ be yea :-D - I figure it's probably worth keeping in case something goes awry elsewhere. 

Maybe I should throw an error here - the upcoming distributed error reporting could come in handy here because we'd hear about it on Sentry (IIRC from the presentation this week).